### PR TITLE
Added steps for configuring custom namespace permissions

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/519

Adding documentation based on @andriisoldatenko's docs at https://www.notion.so/astronomerio/Platform-without-ClusterRole-access-ecfd3a0344954075bb546f48ea1e33fc

### CODE REVIEW

File: enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md [hunk 1]
```diff
@@ -72,3 +72,173 @@ Alternatively, when you run `astro deployment create` command via the Astronomer
 If no namespaces are available, you will receive an error when creating a new Deployment in both the UI and the CLI. To reuse a pre-created namespace, you need to first delete the Deployment using it.
 
 > **Note:** Pre-created namespaces may take several days to become available again after deleting their associated Deployment. To immediately reuse a pre-created namespace, you need to hard-delete its associated Deployment as described in [Delete a Deployment](/docs/enterprise/v0.26/deploy/configure-deployment#delete-a-deployment).
+
+## Customize Namespace Permissions
+
+You can create a custom set of permissions for the Astronomer Commander to access your pre-created namespaces. To do so:
```
Comment (jwitz) [inline enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md:78]: What's the "purpose" of this feature/ why would organizations want to use it? Is this essential setup for precreated namespaces?

File: enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md [hunk 2]
```diff
@@ -72,3 +72,173 @@ Alternatively, when you run `astro deployment create` command via the Astronomer
...
+      clusterRoles: false
+    ```
+
+2. Created a new namespace manually:
+
+    ```yaml
+    kubectl create namespace <namespace-name>
+    ```
+
+3. In a new file named `commander.yaml`, grant permissions for the new namespace to Commander.  You can use the following template as a reference:
```
Comment (jwitz) [inline enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md:94]: Is this the exact permissions that someone needs, or just a recommendation/ template?

File: enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md [hunk 3]
```diff
@@ -72,3 +72,173 @@ Alternatively, when you run `astro deployment create` command via the Astronomer
...
+      name: astronomer-commander # Should match service account name, above
+    ```
+
+4. Apply the file to your new namespace:
+
+    ```yaml
+    kubectl apply -f commander.yaml
+    ```
+
+5. To confirm that the configuration was successful, create a new Deployment with a [custom release name](/docs/enterprise/v0.26/deploy/configure-deployment#customize-release-names) in your custom namespace. If you check the logs of the Commander pod running in your custom namespace, you should see that Commander has immediately begun installing Airflow charts for the new Deployment:
```
Comment (jwitz) [inline enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md:239]: What's different about this workflow compared to not using the custom permissions?

File: enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md (hunk 2 above)
Comment (andriisoldatenko) [reply enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md:94]: it's **exact** one.

**Review by andriisoldatenko — APPROVED** [review]

File: enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md [hunk 4]
```diff
@@ -72,3 +72,181 @@ Alternatively, when you run `astro deployment create` command via the Astronomer
...
+
+If your organization uses pre-created namespaces and you want to limit Astronomer's permissions to individual namespaces, you can create an alternative [`Role`](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-example) for the Astronomer Commander service and apply this role to each pre-created namespace individually.
+
+### Prerequisites
+
+This setup assumes that you have:
+
+- An Astronomer installation.
+- A precreated Kubernetes namespace [configured for Astronomer](/docs/enterprise/v0.26/manage-astronomer/pre-created-namespaces#enable-pre-created-namespaces).
+- kubectl.
```
Comment (paolaperaza) [inline enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md:88]: Link to kubectl

File: enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md [hunk 5]
```diff
@@ -72,3 +72,181 @@ Alternatively, when you run `astro deployment create` command via the Astronomer
...
+The Astronomer Commander service is responsible for provisioning resources to your Kubernetes clusters. By default, the Commander service uses a [`ClusterRole`](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#clusterrole-example) to access pre-created namespaces, meaning that it has the same permissions for all namespaces on your cluster.
+
+If your organization uses pre-created namespaces and you want to limit Astronomer's permissions to individual namespaces, you can create an alternative [`Role`](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-example) for the Astronomer Commander service and apply this role to each pre-created namespace individually.
+
+### Prerequisites
+
+This setup assumes that you have:
+
+- An Astronomer installation.
+- A precreated Kubernetes namespace [configured for Astronomer](/docs/enterprise/v0.26/manage-astronomer/pre-created-namespaces#enable-pre-created-namespaces).
```
Comment (paolaperaza) [inline enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md:87]: Standard spelling of `pre-created`

File: enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md [hunk 6]
```diff
@@ -72,3 +72,181 @@ Alternatively, when you run `astro deployment create` command via the Astronomer
...
+      apiGroup: rbac.authorization.k8s.io
+      kind: Role
+      name: deployment-commander-role # Should match name of Role
+    subjects:
+    - namespace: astronomer # Should match namespace where SA lives
+      kind: ServiceAccount
+      name: astronomer-commander # Should match service account name, above
+    ```
+
+3. Apply the file to your new namespace:
```
Comment (paolaperaza) [inline enterprise/v0.26/06_manage-astronomer/11_pre-create-namespaces.md:241]: New? or precreated?

**Review by paolaperaza — APPROVED** [review]
LGTM!

**Pull request merged**